recipe-server/normandy/settings.py

```diff
@@ -185,7 +186,7 @@ def MIDDLEWARE(self):
     def LOGGING(self):
         return {
             'version': 1,
-            'disable_existing_loggers': True,
+            'disable_existing_loggers': False,
```

Comment:
This makes logging while running in Gunicorn work correctly, which helped greatly in debugging this.